tests/importer/importer.cpp

```diff
@@ -50,16 +50,19 @@ TEST(Importer, noWarningDiamondImport)
     EXPECT_TRUE(model->hasUnresolvedImports());
     importer->resolveImports(model, resourcePath("importer/"));
     EXPECT_FALSE(model->hasUnresolvedImports());
+    EXPECT_EQ(size_t(0), importer->errorCount());
 }
 
 TEST(Importer, warningCircularImportReferencesComponent)
 {
     const std::string errorMessage =
         "Cyclic dependencies were found when attempting to resolve a component in the model 'circularImport1'. The dependency loop is:\n"
-        " - component 'i_am_cyclic' is imported from 'c2' in 'circularImport_2.cellml';\n"
-        " - component 'c2' is imported from 'c3' in 'circularImport_3.cellml';\n"
-        " - component 'c3' is imported from 'i_am_cyclic' in 'circularImport_1.cellml'; and\n"
-        " - component 'i_am_cyclic' is imported from 'c2' in 'circularImport_2.cellml'.";
+        " - component 'i_am_cyclic' specifies an import from ':this:' to '"
+        + resourcePath("importer/") + "circularImport_2.cellml';\n"
+                                      " - component 'c2' specifies an import from '"
+        + resourcePath("importer/") + "circularImport_2.cellml' to '" + resourcePath("importer/") + "circularImport_3.cellml'; and\n"
+                                                                                                    " - component 'c3' specifies an import from '"
+        + resourcePath("importer/") + "circularImport_3.cellml' to '" + resourcePath("importer/") + "circularImport_1.cellml'.";
```

Comment:
Replace with:
```c++
std::string("Cyclic dependencies were found when attempting to resolve a component in the model 'circularImport1'. The dependency loop is:\n")
+ " - component 'i_am_cyclic' specifies an import from ':this:' to '" + resourcePath("importer/") + "circularImport_2.cellml';\n"
+ " - component 'c2' specifies an import from '" + resourcePath("importer/") + "circularImport_2.cellml' to '" + resourcePath("importer/") + "circularImport_3.cellml'; and\n"
+ " - component 'c3' specifies an import from '" + resourcePath("importer/") + "circularImport_3.cellml' to '" + resourcePath("importer/") + "circularImport_1.cellml'.";
```
to make it much easier (for me at least!) to read.

Reply:
Clang format says no.  I tried that already.

Reply:
does the `std::string` make a difference for clang format here? rather than not having it.

Reply:
> Clang format says no. I tried that already.

I have tested the above code with ClangFormat 12 and it's all fine.

> does the `std::string` make a difference for clang format here? rather than not having it.

Don't know. All I know is that `std::string` is needed otherwise the compiler complains about us trying to add two `char*`.